public/app/features/alerting/unified/components/rule-editor/FolderAndGroup.tsx

```diff
@@ -98,8 +99,8 @@ export function FolderAndGroup({
 
   const styles = useStyles2(getStyles);
 
-  const folder = watch('folder');
-  const group = watch('group');
+  const [folder, group, type] = watch(['folder', 'group', 'type']);
+  const isGrafanaRecordingRule = type === RuleFormType.grafanaRecording;
```

Comment:
We have utils for this, don't we? Could we use `isGrafanaRecordingRuleByType` here/similar where necessary?

Reply:
💯